e2e/react-router/basic-file-based-code-splitting/src/routes/viewport-test.tsx

```
@@ -1,3 +1,4 @@
-export const Route = createFileRoute({
+import { createFileRoute } from '@tanstack/react-router'
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)